rest-api-spec/src/main/resources/rest-api-spec/test/indices.put_alias/all_path_options.yml

```diff
@@ -27,9 +27,9 @@ setup:
       indices.get_alias:
         name: alias
 
-  - match: {test_index1.aliases.alias: {}}
+  - match: {test_index1.aliases.alias: { is_write_index: false }}
```

Comment:
why  are these false? this is the only index in this alias so it should be true?